openmetadata-ui/src/main/resources/ui/src/components/BotsDetail/BotsDetail.component.tsx

```diff
@@ -293,11 +293,22 @@ const BotsDetail: FC<BotsDetailProp> = ({
 
   const getBotsTokenExpiryDate = () => {
     if (botsTokenExpiry) {
+      // get the current date timestamp
+      const currentTimeStamp = moment().valueOf();
```

Comment:
```suggestion
      const currentTimeStamp = Date.now();
```

Reply:
Let's not leverage more on `moment` as they are no longer provided support. Check [here](https://momentjs.com/)
Will start eliminating use of moment as and when we touch the code. Moment is no longer maintained.
Also I will encourage you to use libs only if same things are not achieved via core JS concepts.

Reply:
Ahh, Got it, Thanks @chirag-madlani.